src/dto/output/alias.rs

```diff
@@ -4,7 +4,7 @@
 use bee_message_stardust::output as stardust;
 use serde::{Deserialize, Serialize};
 
-pub use super::{
+pub(crate) use super::{
```

Comment:
I don't know why this was `pub` in the first place.

```suggestion
use super::{
```